### ISSUE TITLE
Deleting plugin never completes

### ISSUE THREAD

**Issue by ursomniac** [divio/django-cms#5477]
django-cms==3.2.5;  Django==1.8.13

If I have a page with plugins and I attempt to delete one from the page (either through the "Structure" UI or double-clicking on the plugin on the "Content" view of the page,  it never completes (just spins forever).

It appears to happen to any plugin (custom-built ones but also the "native" text, etc. plugins).

Any ideas?   The site is using MySQL==5.6.17, but it doesn't appear to be a hang there (I've checked "SHOW FULL PROCESSLIST" and there's nothing "stuck").     It's not a server issue (I can reproduce it on different machines).   

Any suggestions of where to look next would be appreciated.

**Comment by vxsx** [comment]
Any chance it is a duplicate of https://github.com/divio/django-cms/issues/4376 ?
tl;dr the can happen if you use firefox with 1password/lastpass extensions enabled

If it's not the case then I'd love to hear more details. OS / Browser / any javascript errors in the browser console?

**Comment by piotrjakimiak** [comment]
Does it and with 500 error?
It is ajax request. Use developer tools in Your browser and check request result .

**Comment by ursomniac** [comment]
vxsx ---- OMG - I think you've hit it.   I AM using Firefox and just recently installed that extension.  Lemme check to see if that is the culprit.

**Comment by ursomniac** [comment]
YES - that's the problem.
